ReflectometryServer/footprint_calc.py

```diff
@@ -106,7 +109,7 @@ def distance(self, comp1, comp2):
 
         Returns: The distance between the two components in mm
         """
-        assert comp1, comp2 in self.setup.positions.keys()
+        assert comp1, comp2 in list(self.setup.positions.keys())
```

Comment:
In this assert, `comp1` is the condition and `comp2 in list(self.setup.positions.keys())` is the message that it will give if assertion is raised. I believe this is not what we are trying to achieve here.

Reply:
good catch!

Reply:
On reflection, I  have removed this line as I don't think it does anything useful (keyerrors get caught at the top level in the `FootprintManager`)